### ISSUE TITLE
[Bug] fixed82num not accepting empty string as valid

### ISSUE THREAD

**Issue by snowypowers** [CityOfZion/neon-js#105]
Right now, the util method `fixed82num` does not accept an empty string as a valid input. That is a valid train of thought.

However, NEO nodes truncate their fixed8 strings when they send it over by cutting off the trailing zeroes. For example, 0.00000020 tokens is received as `14` and not the full `1400000000000000`

Following this train of thought, the base-10 number `0` will be translated and truncated as `''` (empty string) by NEO node.

TLDR: NEO node gives us empty string for fixed8 ZERO. We should recognise that.

@notatestuser  FYI since this was your contributions. Would be good to hear your thoughts but I will accelerate this patch for now.

**Comment by notatestuser** [comment]
Great that you found this, feel free to patch it up 👍
